public/app/plugins/datasource/testdata/QueryEditor.tsx

```diff
@@ -46,10 +46,6 @@ export const QueryEditor = ({ query, datasource, onChange, onRunQuery }: Props)
     onRunQuery();
   };
 
-  useEffect(() => {
-    onUpdate(query);
-  }, []);
-
```

Comment:
Hmm, strange, I remember the graphs weren't rendered properly on page reload after saving dashboard, so that's why I added this call on mount, but I just checked it again and it works without it too.